### DISCUSSION TITLE
BT-13713-LotResult  notice subtype 25

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#415] [Q&A] [answered]
Good Morning Lab !
in Notice subtype 25 if you do not set BT-13713-LotResult in the form you get an xsd Validation error as it is the unique mandatory field 
<xsd:complexType name="LotResultType">
		<xsd:sequence>
                         *************************************************************************************************
			<xsd:element ref="efac:TenderLot" minOccurs="1" maxOccurs="1"/>
		</xsd:sequence>
What about to set it mandatory in fields.json so that it can checked  before making the xml ?
Bye 
Renato

**Comment by rouschr** [comment]
Maybe @YvesJo knows about this as this is XSD and rule related.

**Comment by YvesJo** [comment, marked as answer]
Hi Renato,
The Business Group Procedure Lot Result is optional for the form 25, marking BT-13713 as mandatory would make this BG mandatory. The existence of this field is conditioned by the presence of this BG which is inherently related to the existence of its technical identifier (OPT-322). Existence of BT-13713 is therefore expressed as CM/CF rules which had to be deactivated with SDK 1.3 and will progressively be reactivated with the next releases.
